webserver/server/app/models/pagination/slotBoundsPagination.sql

```diff
@@ -0,0 +1,56 @@
+/* @name slotBoundsPagination */
+WITH
+    low_block AS (
+        SELECT
+            "Block".id,
+            "Block".slot
+        FROM
+            "Block"
+        WHERE
+            slot <= :low! AND tx_count > 0
```

Comment:
I don't think so. This is used later as the `:after_tx_id!` (when the pagination after parameter is not present). And that  parameter is used as `"Transaction".id > :after_tx_id!`. 

Also, the lower bound of the slot range is supposed to be exclusive. Although I've been actually asking myself if this was a good idea or not.

So if you ask for 1 as the lower bound, potentially the first tx needs to be at least in slot 2. If we put `>= :low!` here and there is no block in slot 1, it would find the last tx in slot 2, and we would end up skipping it because of the other condition.

Reply:
Probably worth adding a comment to explain this